platform/feature-api/communication/port-authority/src/tests/_lifecycles.ts

```diff
@@ -35,9 +35,9 @@ export function bubblewrapLifecycle(
 ): PortSpecLifecycle {
     return async () => {
         const ports = await l<Uint8Array>();
-        return mapResource(ports, ([send, recv]) => {
+        return mapResource(ports, ([sent, recv]) => {
```

Comment:
How I read this code, it is the "send port", not the "sent port", so it should stay as it was, hm?